buildworker.rst

```diff
@@ -1,21 +1,21 @@
 
-.. _buildslave:
+.. _buildworker:
 
-Running a buildslave
-====================
+Running a buildworker
```

Comment:
"buildworker" doesn't really fit the terminology now used by Buildbot and doesn't flow as easily as "buildslave", either.  Just "worker" fits Buildbot's new terminology, with "buildbot worker" for disambiguation where necessary.  This applies pretty much everywhere in this PR, and it's a judgement call as to whether each instance needs disambiguation.